client/my-sites/comments/comment-list/index.jsx

```diff
@@ -168,6 +187,27 @@ export class CommentList extends Component {
 						/>
 					) }
 				</ReactCSSTransitionGroup>
+
+				<ReactCSSTransitionGroup
+					transitionEnterTimeout={ 300 }
+					transitionLeaveTimeout={ 150 }
+					transitionName="comment-list__transition"
+				>
+					{ null === filteredComments &&
+						<CommentDetailPlaceholder
+							key="comment-detail-placeholder"
+						/>
+					}
+
+					{ 0 === size( filteredComments ) &&
+						<EmptyContent
+							illustration="/calypso/images/comments/illustration_comments_gray.svg"
+							illustrationWidth={ 150 }
+							key="comment-list-empty"
+							{ ...this.getEmptyMessage() }
```

Comment:
You can tie this spot in with my comment above. Actually down here I find it a bit unclear because I'm left wondering what types of data are being passed in as props...